plugins/src/main/java/com/google/firebase/gradle/plugins/DackkaPlugin.kt

```diff
@@ -168,9 +177,11 @@ abstract class DackkaPlugin : Plugin<Project> {
   }
 
   private fun registerGenerateDackkaDocumentationTask(
-    project: Project
+    project: Project,
+    kotlindocOnly: Boolean,
```

Comment:
If what you need to know whether it should publish kotlindoc only is in project, why don't you get it from it here and instead pass an extra argument?